src/app/common/stores/applications/Faucet/Wallet.store.ts

```diff
@@ -80,7 +80,13 @@ class WalletStore extends BaseStore implements Wallet {
         const wallet = update[0];
         const address = update[0]?.accounts[0]?.address;
         await this.walletHandler(wallet);
-        await this.networkHandler(networkId);
+        if (Number(networkId) !== NETWORKS.MAINNET) {
+          this.wrongNetwork = false;
```

Comment:
isWrongNetwork

Reply:
we have another getter which calls isWrongNetwork and he is return wrongNetwork field